src/index.js

```diff
@@ -212,6 +229,43 @@ function readFile(fileNames) {
     })    
 }
 
+
+function readFileIgnore(fileNames) {
```

Comment:
Suggestion: Instead of reading the ignore file and regular file with `-i` option, you can use  `-i` to get the ignore URLs and use my exitsting `-f` option to read the file after. 

``` 
fbl -i ignoreurls.txt -f test.txt
```